app/scripts/views/articles/list.js

```diff
@@ -7,6 +7,6 @@ import { props } from 'scripts/decorators';
 })
 export default class ArticlesListView extends Marionette.CollectionView {
   childViewOptions() {
-    return { page: this.options.page }
+    return { page: this.options.page };
```

Comment:
> ECMAScript 6 coders nowadays can get rid of nearly all semicolons and remove clutter from their source code. Ralf  S. Engelschall is a strong promoter of reducing source code to its bare minimum. Hence, in his personal opinion
ECMAScript 6 should be coded with as less syntactic sugar as possible and hence without semicolons. 

what do you think about getting rid of unnecessary semicolons?